pyramid/config/views.py

```diff
@@ -1597,8 +1597,8 @@ def add_notfound_view(
         ):
         """ Add a default :term:`Not Found View` to the current configuration
         state. The view will be called when Pyramid or application code raises
-        an :exc:`pyramid.httpexceptions.HTTPNotFound` exception (e.g., when a
-        view cannot be found for the request).  The simplest example is:
+        an :exc:`pyramid.httpexceptions.HTTPTemporaryRedirect` exception (e.g.,
```

Comment:
This is wrong... this should not be changed from HTTPNotFound.

Reply:
As I understand it this PR should only be changing the default for the `append_slash=True` behavior by modifying the `AppendSlashNotFoundViewFactory`'s `redirect_class` arg. This docs change does not make sense.

Reply:
You're obviously right, it was too late on Friday :)